app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -636,6 +640,9 @@ ListingService = ($http, $localStorage, $modal, $q, $state, $translate) ->
     if opts
       Service.stubPreferences(opts)
 
+  Service.stubListingFeatures = ->
+    Service.listing.STUB_CommunityRestriction = 'All People'
```

Comment:
@chrisdolendo @catcall re: your earlier comment, the "All People" comes from the way they said it will eventually be set up in Salesforce, with "All People" or "One Person" being the values: https://www.pivotaltracker.com/n/projects/1397652/stories/139237931